Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -143,3 +144,59 @@ TEST_CASE(test_ppm)
     auto frame = ppm.frame(0).release_value_but_fixme_should_propagate_errors();
     EXPECT(frame.duration == 0);
 }
+
+TEST_CASE(test_targa_bottom_left)
+{
+    auto file = Core::MappedFile::map("/res/html/misc/targasuite_files/oh-lena-bottom-left-uncompressed.tga"sv).release_value();
+    auto tga = Gfx::TGAImageDecoderPlugin((u8 const*)file->data(), file->size());
+    EXPECT(tga.frame_count());
+
+    EXPECT(tga.sniff());
+    EXPECT(!tga.is_animated());
+    EXPECT(!tga.loop_count());
+
+    auto frame = tga.frame(0).release_value_but_fixme_should_propagate_errors();
```

Comment:
Instead of using the `fixme` method, wouldn't it be more appropriate to explicitly check for errors?